docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+
+[discrete]
+[[threat-match-detected-section]]
+==== Threat match detected
+
+The Threat match detected section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
+
+[discrete]
+[[enriched-with-threat-intelligence-section]]
+==== Enriched with threat intelligence
+
+The Enriched with threat intelligence section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. 
+
+Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected time frame, the section displays a message that none are available.
+
+NOTE: The event enrichment query uses the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
+
+[discrete]
+[[correlations-section]]
+=== Correlations 
+
+The Correlations sub-section shows how an alert is related to other alerts and offers ways to investigate related alerts. Use this information to quickly find patterns between alerts and then take action. 
 
-All other available threat intelligence data appears in the **Enriched with Threat Intelligence** subsection.
+Click **Correlations** to expand the **Correlations** sub-tab in the left panel. The sub-tab displays the following information: 
+
+* **Suppressed alerts**: Indicates that the alert was created with alert suppression, and shows how many duplicate alerts were suppressed. This section only appears if alert suppression is enabled for the rule.
+* **Alerts related by source event**: Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the **Investigate in timeline** button to examine related alerts in Timeline.
+* **Cases related to the alert**: Shows the total number and names of cases to which the alert has been added. Click a case’s name to open its details.
+
+If you have a https://www.elastic.co/pricing[Platinum or higher subscription], these details are also included:
+
+* **Alerts related by session ID**: Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the **Include session data** setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Comment:
i don't think related by session is guarded by license @PhilippeOberti is that correct? though it could be confusing that session view does require platinum license

Reply:
In our code, related by session is not protected by any license check. I do not know if that is the expected behavior though...

Reply:
Thanks, both! Let me know if the line about license reqs needs to be removed. This content was copied from the old flyout docs so it reflects the way that data for this section was handled/displayed in the old flyout.

Reply:
On that note, a good amount of the content in this draft was repurposed from the old alert details flyout docs. My general understanding of the flyout redesign was that the UI and flows had changed, but the pre-reqs for using certain feature within the flyout hadn't.

Reply:
Confirmed with @christineweng via Slack that data for alerts related by session is available with a basic license in 8.10. Alerts related by process ancestry data is still only available for users with a plat or higher license.

Reply:
Addressed in [92e7483](https://github.com/elastic/security-docs/pull/3816/commits/92e7483737ae677dde652ff744e29966975d4730).